lib/CarControl/CarControl.cpp

```diff
@@ -178,7 +183,7 @@ void CarControl::adjust_paddles(int seconds) {
   ads_max_acc = 0;
   ads_max_dec = 0;
 
-  int cycles = (seconds * 10);
+  int cycles = (seconds);
```

Comment:
I think the `()` can be removed. Not sure if `seconds == cycles`, most likely not with a frequency of 240Mhz..

Reply:
done: removed the not necessary seconds